apps/examples/wc-next10/ButtonSizes.js

```diff
@@ -1,13 +1,5 @@
 import React from 'react';
-import { createComponent } from '@lit-labs/react';
-import { PieButton, BUTTON_SIZE } from '@justeattakeaway/pie-button';
-
-const PieBtn = createComponent({
-    tagName: 'pie-button',
-    elementClass: PieButton,
-    react: React,
-    events: { onCustomEvent: 'CustomEvent' },
-});
+import { PieButtonReact, BUTTON_SIZE } from '@justeattakeaway/pie-button';
```

Comment:
@justeattakeaway/ui-reviewers-design-system, not sure if we've discussed this yet did we consider the following format for the imports?

`import { PieButton } from '@justeattakeaway/react/pie-button';`

I believe the above format is more semantic to me and will allow us to divide and compose wrappers in a more structured way. for example, we can have a wrapper for:
`import { PieButton } from '@justeattakeaway/vue/pie-button';`
`import { PieButton } from '@justeattakeaway/angular/pie-button';`

instead of postfixing the framework name 👀 

Alternatively, we can ship another package that wraps our components and automates the generation like the current approach of [Adobe](https://opensource.adobe.com/spectrum-web-components/using-swc-react/).

Reply:
I do like the first approach, won't that require a lot of restructuring? I like the idea of something like this -

```
import { PieButton } from '@justeattakeaway/pie-button/react';
import { PieButton } from '@justeattakeaway/pie-button';
```

But I tried this and struggled to get it to work and compile - if you have time to pair, happy to give it another go! We sort of discussed this here through using exports - https://github.com/justeattakeaway/pie/pull/426#discussion_r1188627708

I personally don't think the use of another package is optimal just because it's another separate thing to update and publish. I'm not sure it's necessary if we can do it within the component - but maybe I'm just oversimplifying it 😄

Reply:
I think the format @LTurns mentioned would be my preferred import syntax (and that fits directly with how NPM lets you extend packages). It might be good to try and pair someone (likely with React experience I'd guess) to see if it's possible to get that syntax working or if it's just not possible.

I think if we can do it in one component that feels a bit cleaner (to me anyway) – but it could be that we wrap all of our components into a React bundle potentially that includes the wrappers.

Interested to hear others thoughts on this though too – /cc @fernandofranca @xander-marjoram @jamieomaguire

Reply:
managed to get this working with the import thanks to @siggerzz for the extra help! It currently works as:

```
import { PieButton } from '@justeattakeaway/pie-button/dist/react';
import { PieButton } from '@justeattakeaway/pie-button';`
```
We reference it through the dist, but much better than changing the name 😄